packages/app-layout/src/vaadin-app-layout-mixin.js

```diff
@@ -508,8 +503,11 @@ export const AppLayoutMixin = (superclass) =>
     /** @protected */
     _blockAnimationUntilAfterNextRender() {
       this.setAttribute('no-anim', '');
-      afterNextRender(this, () => {
-        this.removeAttribute('no-anim');
+
+      requestAnimationFrame(() => {
```

Comment:
I wonder if it would make sense to use a debouncer here instead since this method is used in `_resize` which can be triggered repeatedly when resizing the window.

Reply:
Good point, I'll add the debouncer.